microcosm_pubsub/context.py

```diff
@@ -27,43 +26,39 @@ def __init__(self, graph):
         self.enable_ttl = graph.config.sqs_message_context.enable_ttl
         self.initial_ttl = graph.config.sqs_message_context.initial_ttl
 
-    def __call__(self, message, **kwargs):
+    def __call__(self, context, **kwargs) -> Dict[str, str]:
```

Comment:
The `SQSMessageContext` now supports passing in an `SQSMessage` directly. 

This works as long as we don't also invoke using a `dict`, which we do in two places. I'll point those out and fix in a future PR.